src/clj/puppetlabs/services/versioned_code_service/versioned_code_core.clj

```diff
@@ -1,6 +1,7 @@
 (ns puppetlabs.services.versioned-code-service.versioned-code-core
   (:require [schema.core :as schema]
             [clojure.tools.logging :as log]
+            [puppetlabs.puppetserver.common :refer [CodeId code-id-validation-error-msg Environment]]
```

Comment:
Generally I think we try to avoid doing `:refer` outside of tests, so we can more easily tell where things are coming from when reading through code